std_msgs/Bool doTrack
std_msgs/Int16 trackMode 
std_msgs/Bool depthType
